source/_integrations/vacuum.markdown

```diff
@@ -3,6 +3,7 @@ title: Vacuum
 description: Instructions on how to setup and use vacuum's in Home Assistant.
 ha_release: 0.51
 ha_domain: vacuum
+ha_iot_class: Local Polling
```

Comment:
This is a entity component, which is used to build other integrations on. The `vacuum` integration by itself, cannot be used. Hence it cannot have an IoT class.

```suggestion
ha_iot_class: ~
```